### DISCUSSION TITLE
Preset `config:js-app` triggers warnings for node

### DISCUSSION THREAD

**Discussion by PeterRist** [renovatebot/renovate#23245] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us what version of Renovate you run.

36.1.11

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the problem

The `renovate.json` looks like
```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": ["config:js-app"]
}
```

I just updated from version v35 to v36 and receiving some warnings.
```
 "repoProblems": ["WARN: Use matchDepNames instead of matchPackageNames"]
```

```
{
                 "depType": "engines",
                 "depName": "node",
                 "currentValue": ">=18",
                 "datasource": "github-tags",
                 "packageName": "nodejs/node",
                 "versioning": "node",
                 "commitMessageTopic": "Node.js",
                 "prettyDepType": "engine",
                 "updates": [],
                 "warnings": [],
                 "sourceUrl": "https://github.com/nodejs/node",
                 "registryUrl": "https://github.com/",
                 "currentVersion": "v20.4.0"
               },

```

I've created a complete new repository with the bare-minimum to reproduce this issue on GitHub: https://github.com/PeterRist/renovate-v36-node-warnings/actions/runs/5487589724/jobs/9999364621

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by rarkins** [comment, marked as answer]
Please test with 36.1.14 or above (see #23213)
